workflows/execute_workflows.sh

```diff
@@ -119,7 +123,7 @@ do
         cp "$WORKFLOW" "$WS_DIR"
     done
 
-    DIR_NAME=$(echo $WS_DIR | rev | cut -d'/' -f 2 | rev)
+    DIR_NAME=$(dirname $WS_DIR)
```

Comment:
This does not yield the same result as the former code.
Should be something like `16_ant_complex`
Yields `/some/path/workflows/workspaces`

Reply:
> `16_ant_complex`

What is the `$WS_DIR` in this case? I am sure there's a more readable way to get the same results.

Reply:
Probably `$(basename "$WS_DIR")`

Reply:
`$WS_DIR` is something like `/some/path/to/quiver/workflows/workspaces/16_ant_simple/`. And I'd be glad if we found something more readable :)

Reply:
`$(basename "$WS_DIR")` works fine

Reply:
:+1: